packages/main/src/WheelSlider.js

```diff
@@ -67,6 +67,10 @@ const metadata = {
 		_items: {
 			type: Object,
 		},
+
+		cyclic: {
+			type: Boolean,
```

Comment:
Add jsdoc for this property, it is public

Reply:
Done